pyopae/test_pyopae.py

```diff
@@ -309,6 +309,18 @@ def test_allocate(self):
         ba = bytearray(buff1)
         assert ba[0] == 0xaa
 
+    def test_conext_release(self):
+        assert self.handle
+        self.handle.close()
+        with opae.fpga.open(self.toks[0]) as h:
+            buff = opae.fpga.allocate_shared_buffer(h, 4096)
```

Comment:
what is the difference between the pyshared_buffer.cpp file "shared_buffer_allocate" and the "opae.fpga.allocate_shared_buffer" here

Reply:
opae.cpp associates the module method `allocate_shared_buffer` with the function, `shared_buffer_allocate`, defined in pyshared_buffer.h|cpp